### DISCUSSION TITLE
Cannot backup a single video

### DISCUSSION THREAD

**Discussion by PilaScat** [immich-app/immich#6725] [Q&A]
### The bug

I installed immich 2 months ago, it backed up all files except a single video, I don't understand why, maybe it's because it was done at the turn of the year (23-24)?

### The OS that Immich Server is running on

unRAID 6.12.6

### Version of Immich Server

v1.93.3

### Version of Immich Mobile App

v1.93.3

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Running on docker with imagegenius/immich:latest image plus redis and postegres shared between my projects (yes tensorchord/pgvecto-rs)
```

### Your .env content

```Shell
Default, nothing particular
```

### Reproduction steps

```bash
1. Did the video
2. Not syncing
```

### Additional information

Video desc
![Screenshot_20240129_152010_Gallery](https://github.com/immich-app/immich/assets/12989749/96416adf-dbda-4bd4-a87c-4daa132593c0)
Error from immich (can't find relevant logs)
![Screenshot_20240129_151722_One UI Home](https://github.com/immich-app/immich/assets/12989749/3857b901-3673-4c81-a3a7-34d2573be4aa)

**Comment by alextran1502** [comment]
Using latest tag generally a bad idea because it could be tag used for development. Try to pin it to a specific version

**Comment by martabal** [comment]
> Using latest tag generally a bad idea because it could be tag used for development. Try to pin it to a specific version

He's using the AIO image. The `latest` tag is just an alias for the latest Immich release

**Comment by PilaScat** [comment]
> Using latest tag generally a bad idea because it could be tag used for development. Try to pin it to a specific version

Isn't the problem, I have tried even reinstalling a specific fixed version but same

**Comment by martabal** [comment]
Is your video corrupted ?

Did you try to upload it on the web app ?

**Comment by alextran1502** [comment]
@PilaScat Are you using a reverse proxy? The video seems large and might be killed by the background process before finishing backing up. 

Can you upload it in the app?

**Comment by PilaScat** [comment]
yes via web in local it works, seems proxy related, any advice?
already have set max body size to 5000M

**Reply by aviv926** [reply]
Do you have any logs from the proxy?

**Reply by adambeck7** [reply]
@PilaScat are you using Cloudflare in any capacity?

**Comment by PilaScat** [comment]
> @PilaScat are you using Cloudflare in any capacity?

Yes, I'm using cloudflare argo tunnel, at home is reverse proxied the same and it works

**Reply by adambeck7** [reply]
If it works on local but not via web, it's almost certainly cloudflare's file size limit on the free tier. It doesn't matter what you bump your local proxy to if cloudflare is still in the mix.

**Reply by adambeck7** [reply]
We really need more info to answer definitively though. Do you have any logs? Post your config files. But I'm pretty confident it's cloudflare because I've hit this same issue a bunch in the past with nextcloud and other apps.

**Reply by PilaScat** [reply]
now I'll make new video and post the logs

**Comment by PilaScat** [comment]
Hi guys, I'm back here after a few months to tell you that the problem is also present in LAN (behind reverse proxy)
I remind you of my nginx configuration:
```
client_max_body_size 0;
proxy_redirect off;
```
This time I'm trying with 2 video, 2gb and 40gb
The first stop uploading at some percentage every time, the other I can't even see, it disappears
[Immich_log_2024-03-24T121045.474594.csv](https://github.com/immich-app/immich/files/14735235/Immich_log_2024-03-24T121045.474594.csv)

**Reply by PilaScat** [reply]
Nextcloud with the same config works, now I'll try to upload from web, proxied and not

**Reply by PilaScat** [reply]
Ok is not a proxy problem, uploaded via ip:port and photos.domain.tld from pc and it works, it's the app that is problematic, another problem is this, the video is 40 minutes long (40gb) but the app tell 0:00 
![image](https://github.com/immich-app/immich/assets/12989749/a50d0a9d-16ad-4b1b-ba74-32410cad4e37)

**Reply by PilaScat** [reply]
@alextran1502
